build/index.html

```diff
@@ -1,15 +1,218 @@
+                <img src="img/astoria-mobile@1x.jpg" srcset="img/astoria-mobile@2x.jpg" alt="Пейзаж круиза “Астория”">
+              </picture>
+              <p class="main-cruise__name">Круиз “Астория”</p>
+            </div>
+            <!-- Открытая карточка -->
+            <div class="main-cruise__card main-cruise__card--reverse">
+              <p class="main-cruise__name">Круиз “Элайн”</p>
+              <dl class="main-cruise__list">
+                <dt class="main-cruise__point">Маршрут</dt>
+                <dd class="main-cruise__description">Аргентина - Фолклендские острова - Антарктида - Чили</dd>
+                <dt class="main-cruise__point">Порт отправления</dt>
+                <dd class="main-cruise__description">Буэнос-Айрес</dd>
+                <dt class="main-cruise__point">Продолжительность</dt>
+                <dd class="main-cruise__description">20 дней</dd>
+                <dt class="main-cruise__point">Цена</dt>
+                <dd class="main-cruise__description">от 14500 $</dd>
+              </dl>
+              <button class="main-cruise__button" type="button">
+                <span>Забронировать</span>
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="main-ordering">
+        <h2 class="visually-hidden">Секция с бронированием</h2>
+        <div class="main-ordering__wrapper">
+          <dl class="main-ordering__list">
+            <dt class="main-ordering__title">Раннее бронирование</dt>
+            <dd class="main-ordering__description">Забронируйте тур за 3-6 месяцев
+              до даты выезда и сэкономьте до 50%
+            </dd>
+          </dl>
+          <div class="main-ordering__container">
+            <p class="main-ordering__text">Оставьте заявку и мы свяжемся с вами в ближайшее время</p>
+            <form class="main-ordering__form" action="https://echo.htmlacademy.ru/" method="post">
+              <p class="main-ordering__group">
+                <label class="visually-hidden" for="form-name">Имя</label>
+                <input class="main-ordering__field" type="text" id="form-name" placeholder="Имя">
+              </p>
+              <p class="main-ordering__group">
+                <label class="visually-hidden" for="form-tel">Телефон</label>
+                <input class="main-ordering__field" type="tel" id="form-tel" placeholder="Телефон">
+              </p>
+              <p class="main-ordering__group">
+                <label class="visually-hidden" for="form-email">Email</label>
+                <input class="main-ordering__field" type="email" id="form-email" placeholder="Email">
+              </p>
+              <button class="main-ordering__button" type="submit">Оставить заявку</button>
+            </form>
+          </div>
+        </div>
+      </section>
+
+      <section class="main-contacts">
+        <h2 class="visually-hidden">Секция с контактами</h2>
```

Comment:
Можно использовать семантический тэг. Просто без заголовка
```suggestion
      <address class="main-contacts">
```